tests/test_marc.py

```diff
@@ -147,7 +147,72 @@ def test_marc_record_all_fields_transform_correctly():
                 kind="singer",
             ),
         ],
+        edition="9th ed. / Nick Ray ... [et al.]. Unabridged.",
         literary_form="Nonfiction",
+        subjects=[
+            timdex.Subject(
+                value=[
+                    "Renoir, Jean, 1894-1979 (DLC)n  79018203 Bibliography. "
```

Comment:
Go was picking up the subfield 0 but I think this clutters the value and we don’t have anywhere to store it separately in the data model. With the local data, take a look at ID 990015001040106761 for an example and let me know what you think (also about the formatting in general, some values look better than others but removing subfield 0 would likely fix a lot)

Reply:
This is interesting and I agree that we should drop $0. I suspect that when we did the original work, we weren't using $0 in practice in Aleph but we're seeing it used more now in Alma and it's indeed not appropriate for this use case. (We may want it for some fancy subject enhancement later, but for now...definitely not).

I'm seeing, in production, stuff like:
```
          "subjects": [
            "Money History 18th century. France",
            "Assignats History.",
            "Money Social aspects History 18th century. France",
            "Money Political aspects History 18th century. France",
            "Monetary policy History 18th century. France",
            "(OCoLC)fst00819080 Assignats.",
            "(OCoLC)fst00901974 Economic history.",
            "(OCoLC)fst01025230 Monetary policy.",
            "(OCoLC)fst01025265 Money.",
            "(OCoLC)fst01025293 Money Political aspects.",
            "(OCoLC)fst01025301 Money Social aspects.",
            "(OCoLC)fst01919741 Politics and government",
            "France History Revolution, 1789-1799.",
            "France Economic conditions 18th century.",
            "France Politics and government 1789-1799.",
            "(OCoLC)fst01204289 France."
          ]
```
which is kinda gross. Thanks for noticing this and yes please remove $0!